app/_components/SubmitGuessButton.tsx

```diff
@@ -110,32 +68,29 @@ function SubmitGuessButton({
           body: JSON.stringify(data),
         }
       );
-      // checking the tracking pin
-      for (const pin of pins) {
-        if (pin.poi_id === trackingPin?.poi_id) {
-          trackingPin.is_completed = true;
-        }
-      }
-      console.log("response", response);
+      const { message } = await response.json() as {message: string};
+      console.log(message);
```

Comment:
Do we want this console log here?

We should probably return the `response` here as well so that the client can confirm the `POST` was successful.